crates/oxc_linter/src/globals.rs

```diff
@@ -424,3 +424,22 @@ pub const HTML_TAG: phf::Set<&'static str> = phf_set! {
     "wbr",
     "xmp",
 };
+
+pub const RESERVED_HTML_TAG: phf::Set<&'static str> = phf_set! {
```

Comment:
can we move this to `aria_props.rs` or add some docs for what this mean? - from a glance and without lookling at eslint it's kinda hard to tell

Reply:
Roger that. We will respond!